lib/ui/myplot/new_my_plot_child_item.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:farmsmart_flutter/data/model/crop_entity.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:flutter/material.dart';
+import 'package:farmsmart_flutter/ui/common/network_image_from_future.dart';
+import 'my_plot_page.dart';
+
+class MyNewPlotListItem {
+  Widget buildListItem(CropEntity cropsData, HomeMyPlotPageStyle myPlotStyle) {
```

Comment:
lets add a view model as previous comment (then we can inject mock data externally to the widget and remove any need for mock strings in the widget itself).